cloud/aws/templates/aws_oidc/external_vpc.tf

```diff
@@ -0,0 +1,21 @@
+// File containing the necessary data sources if enable_managed_vpc=false
```

Comment:
**suggestion:** May be helpful to include a comment that enable_managed_vpc will be false if the `external_vpc` variable is set with the expected values.